docs/feature/storage/indexing-and-storage.md

```
@@ -0,0 +1,172 @@
+(indexing-and-storage)=
+(storage-internals)=
+
+# Indexing and storage in CrateDB
+
+:::{article-info}
```

markdownlint-cli2 (0.18.1)

Heading style
Expected: atx; Actual: setext

(MD003, heading-style)